src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -185,7 +192,8 @@ const DateForm: React.FC<{
   setShowError: (showError: boolean) => void
   modal?: boolean
   refetch: () => Promise<void>
```

Comment:
Nitpick, I think this is fine if one or two props, but when we have this many props I think it would be nice to have an Interface DateFormProps, what do you think?

Reply:
I totally agree, I'd rather have a prop type for that

Reply:
Let's do this when splitting this into two files 👍